lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -457,11 +659,13 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
     }
   }
 
-  void _maybeSendAction(html.KeyboardEvent event) {
-    if (_inputConfiguration.inputType.submitActionOnEnter &&
-        event.keyCode == _kReturnKeyCode) {
-      event.preventDefault();
-      _onAction(_inputConfiguration.inputAction);
+  void _maybeSendAction(html.Event event) {
+    if (event is html.KeyboardEvent) {
+      if (_inputConfiguration.inputType.submitActionOnEnter &&
+          event.keyCode == _kReturnKeyCode) {
+        event.preventDefault();
```

Comment:
Why do we need to prevent default? Consider leaving a comment.

Reply:
this is only a format change. you can discuss with @mdebbar the keyboard related features. thanks for the suggestions.

Reply:
@mdebbar let me know if there is any comment you want me to add once I'm around the file :)